### PR TITLE
Added Note on the next article or video to watch

### PR DESCRIPTION
**Pull request by hmbrown64**
Added:

{% info %}
For information on sending Campaigns, visit our [Sending an Email](https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/getting_started.html) article or [view the next video](https://vimeo.com/139274837) in our Getting Started Series.
{% endinfo %}

**Description of the change**:
**Reason for the change**:
**Link to original source**:

@ksigler7

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/docs?pullRequest=2836) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/docs?pullRequest=2836) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/docs?pullRequest=2836) it.</sub>

**Comment by hmbrown64** [comment]
Completed for this request: https://github.com/sendgrid/docs/issues/2829

File: source/User_Guide/Marketing_Campaigns/templates.md [hunk 1]
```diff
@@ -79,3 +79,6 @@ Creating a Template from a Campaign
 1. From the left-hand navigation, select **Marketing Campaigns**, and then click **Campaigns**.
 1. Find the campaign you would like to use, click the **action menu**, and then select **Create Template**. 
 
+{% info %}
+For information on Sending Campaigns, visit our [Sending an Email](https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/getting_started.html) article or [view the next video](https://vimeo.com/139274837) in our Getting Started Series.
```
Comment (Whatthefoxsays) [inline source/User_Guide/Marketing_Campaigns/templates.md]: Change to:
For more information, see [Sending an Email] or [view the next video] in our Getting Started series.

**Comment by ksigler7** [comment]
hey @hmbrown64, check out Carlys comments!

**Comment by hmbrown64** [comment]
Submitted changes: https://github.com/sendgrid/docs/pull/2836/commits/07178f538cd8a1f736e4c0916fb88bc7e4008f3a

**Comment by ksigler7** [comment]
added a commit to solve the merge conflict!

**Comment by thinkingserious** [comment]
Hello @hmbrown64,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**